backend/src/jwt/decode.js

```diff
@@ -1,11 +1,12 @@
 import jwt from 'jsonwebtoken'
+import CONFIG from './../config'
 
 export default async (driver, authorizationHeader) => {
   if (!authorizationHeader) return null
```

Comment:
> [<img alt="appinteractive" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/appinteractive) **Authored by [appinteractive](https://github.com/appinteractive)**
_<time datetime="2019-05-07T08:39:59Z" title="Tuesday, May 7th 2019, 10:39:59 am +02:00">May 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -7,7 +7,7 @@ export default async (driver, authorizationHeader) => {
   try {
     const decoded = await jwt.verify(token, process.env.JWT_SECRET)
     id = decoded.sub
-  } catch {
+  } catch (e) {
```


Just as a consistency hint: e stands often for event, try to use err as an error shorthand.